pgjdbc/src/test/java/software/aws/rds/jdbc/postgresql/ca/AwsIamAuthenticationIntegrationTests.java

```diff
@@ -52,6 +54,7 @@ private Properties createProperties(final String user, final String password) {
       properties.setProperty(PGProperty.PASSWORD.getName(), password);
     }
     properties.setProperty(PGProperty.USE_AWS_IAM.getName(), Boolean.TRUE.toString());
```

Comment:
`USE_AWS_IAM` has no use anymore. Could you remove all references to it & update docs too?